### PR TITLE
switch to vndr

### PR DESCRIPTION
**Pull request by runcom**
vndr is almost exactly the same as our old good hack/vendor.sh. Except
it's cleaner and it allows to re-vendor just one dependency if needed
(which we do a lot for containers/image).

close #135

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@mtrmac PTAL - from now on, there would be no need to run hack/vendor.sh to vendor _every_ dependency. Instead `vndr github.com/containers/image` should just vendor c/image instantly (since it's tiny).

File: hack/vendor.sh [hunk 1]
```diff
@@ -1,46 +1,15 @@
...
-rm -rf vendor/
-source 'hack/.vendor-helpers.sh'
+# This file is just wrapper around vndr (github.com/LK4D4/vndr) tool.
+# For updating dependencies you should change `vendor.conf` file in root of the
+# project. Please refer to https://github.com/LK4D4/vndr/blob/master/README.md for
+# vndr usage.
 
-clone git github.com/urfave/cli v1.17.0
-clone git github.com/containers/image master
-clone git github.com/opencontainers/go-digest master
```
Comment (mtrmac) [inline hack/vendor.sh]: This is the only item which has not been replicated in the new `vendor.conf`, and it does seem necessary:

`rm -rf /vendor/*/; vndr` does not recreate `vendor/github.com/opencontainers/go-digest`.

Comment (runcom) [inline hack/vendor.sh]: great catch :+1:

**Comment by runcom** [comment]
> Also, this supersedes #135, doesn’t it?

it does, yes

File: hack/vendor.sh (hunk 1 above)
Comment (runcom) [inline hack/vendor.sh]: fixed

**Comment by mtrmac** [comment]
:+1:

**Pull request merged**